ymap/operators.py

```diff
@@ -61,7 +69,11 @@ def poll(cls, context):
 
     def run(self, context):
         ymap_obj = context.active_object
-        create_ymap_group(sollum_type=SollumType.YMAP_MODEL_OCCLUDER_GROUP, selected_ymap=ymap_obj, empty_name='Model Occluders')
+        ymap_group = create_ymap_group(sollum_type=SollumType.YMAP_MODEL_OCCLUDER_GROUP, selected_ymap=ymap_obj, empty_name='Model Occluders')
+        if ymap_group:
+            bpy.ops.object.select_all(action='DESELECT')
+            ymap_group.select_set(True)
+            context.view_layer.objects.active = ymap_group
```

Comment:
This code seems to be the same in all cases, a bit repetitive, perhaps could be moved into `create_ymap_group`? Possibly behind a bool parameter.